sw/airborne/math/pprz_algebra_float.c

```diff
@@ -618,6 +618,39 @@ void float_quat_of_rmat(struct FloatQuat *q, struct FloatRMat *rm)
   }
 }
 
+#include <stdio.h>
+/**
+ * @brief Tilt twist decomposition of a quaternion (z axis)
+ *
```

Comment:
maybe provide a bit of mathematical details ? it is not so obvious what this function does